src/keystrokehandler.js

```diff
@@ -81,7 +81,7 @@ export default class KeystrokeHandler {
 	 * @param {String|Array.<String|Number>} keystroke Keystroke defined in a format accepted by
 	 * the {@link module:utils/keyboard~parseKeystroke} function.
 	 * @param {Function} callback A function called with the
-	 * {@link module:engine/view/observer/keyobserver~KeyEventData key event data} object and
+	 * {@link module:utils/keyboard~KeystrokeInfo keystroke info} object and
```

Comment:
That's a bit messy now, but it's actually called with `KeyEventData`. The `KeyEventData` objects implements `KeystrokeInfo` but it adds `preventDefault()` and `stopPropagation()` methods (which are mentioned later in this sentence. It'd be good to not rely on engine's object type, so that's why I said it's messy. But other than that, `KeyEventData` is more correct.